bsip-0071.md

```diff
@@ -39,9 +60,12 @@ We now have several choices on handling bad debt:
 
 We need to do a deep review on bad debt before evaluating above solutions.
 
-Bad debt means in some debt positions the debt cannot be fully paid by selling the collaterals via margin call/force settlement at the market price.  In other words, the relevant smartcoin loses sufficient collateral to back the value and will possibly devalue.
+Bad debt means in some debt positions the debt cannot be fully paid by selling the collaterals via margin 
+call/force settlement at the market price.  In other words, the relevant smartcoin loses sufficient collateral 
+to back the value and will possibly devalue.
 
-At this moment, the system should accept the fact that the smartcoin will possibly devalue, find ways to minimize the impact to different parties in the market and the time to revive.
+At this moment, the system should accept the fact that the smartcoin will possibly devalue, 
```

Comment:
It's not just a possibility, but now a fact that the CNY & USD assets have been devalued by BSIP 76.